sr_demos/scripts/sr_demos/demo.py

```diff
@@ -316,10 +325,28 @@ def check_touched_finger(self):
             touched_finger = self.tactiles[0].confirm_touched()
         return touched_finger
 
+    def _enable_touch_detection(self, detect: bool):
+        '''
+            Enables or disables touch detection
+            @param detect - True or False
+        '''
+        for tactile in self.tactiles:
+            tactile.touch_detection = detect
+
+    def touch_detection_enabled(self) -> bool:
+        '''
+            Returns whether touch detection is enabled or not
+            @return True or False
+        '''
+        if not any(tactile.touch_detection for tactile in self.tactiles):
+            return False
+        return True
+
     def stored_states_sequence(self):
```

Comment:
No type hinting?

Reply:
fixed in https://github.com/shadow-robot/sr_interface/pull/823/commits/4d899740e14c6a37b20146403ea06d3b82a837c8